README.md

```diff
@@ -465,6 +465,102 @@ yarn test $adapter/test/unit/my-specific-test.test.ts
 yarn test --watch $adapter/test/unit
 ```
 
+#### Soak Testing
+
+In order to soak test adapters we need to create and push the adapter out to the sdlc cluster. From there we can use the Flux Emulator or K6 to send traffic to it for the amount of time you need.
+
+Prerequisites to starting an external adapter in the sdlc cluster
+
+1. You must be on the vpn to access the k8s cluster.
+2. You must have your kubectx set to the sdlc cluster which also requires you be logged into the aws secure-sdlc account as a user with k8s permissions. To do so it would look something like this but with your specific profile name`aws sso login --profile sdlc`. Instructions to set this up can be found here: https://www.notion.so/chainlink/QA-Kubernetes-Cluster-ca3f1a64e6fd4476ac5a76c8bfcd8624
```

Comment:
What is "kubectx" and how do we set this up?

```suggestion
2. You must have your kubectx set to the sdlc cluster which also requires you be logged into the aws secure-sdlc account as a user with k8s permissions. To do so it would look something like this but with your specific profile name `aws sso login --profile sdlc`. Instructions to set this up can be found here: https://www.notion.so/chainlink/QA-Kubernetes-Cluster-ca3f1a64e6fd4476ac5a76c8bfcd8624
```

Reply:
Good catch. I will update the documentation. It should be kubeconfig or kubernetes context. The instructions in the provided link for setting it up say kubeconfig so I will keep things consistent with that.

fyi `kubectx` is just a tool that simplifies making `kubectl` commands to switch the active kubeconfig between clusters.